apis/r/R/TileDBObject.R

```diff
@@ -28,18 +28,18 @@ TileDBObject <- R6::R6Class(
       }
     },
 
-    #' @description Print the name of the R6 class.
+    #' @description Print the name of the R6 class. [lifecycle: experimental]
```

Comment:
not sure this one is needed if this R's convention for implementing an class accessor